testdata/builtinManifestJsonEx.jsonnet

```diff
@@ -1,22 +1,46 @@
 local a = {
-    foo: "bar",
-    bar: self.foo,
-    baz: 1,
-    bazel: 1.42,
-    boom: -1,
-    bim: false,
-    bam: true,
-    blamo: {
-        cereal: [
-            "<>& fizbuzz",
-        ],
+  foo: 'bar',
```

Comment:
reformatting happened automatically when saving with jsonnetfmt hook, hope that's ok. Let me know if I should revert

Reply:
It's fine. We should probably keep them all formatted (besides the ones intended to test the formatter).